src/pages/Search/components/PreviousButton.tsx

```diff
@@ -0,0 +1,19 @@
+import { Link } from 'react-router-dom';
+import { ReactComponent as PreviousIcon } from 'assets/svg/previous.svg';
+import styles from '../Search.module.scss';
+
+interface Navigate {
+  to? : string;
+}
+
+export default function PreviousButton({ to } : Navigate) {
+  return (
+    <Link to={`${to}`} className={styles['search-nav__button--previous']}>
+      <PreviousIcon title="이전 페이지로 이동" />
+    </Link>
+  );
```

Comment:
defaultProps 는 class component 시절에 주로 사용되었고 
지금은 [default property](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters) 가 나오면서 별로 사용되지 않는것 같아요

```suggestion
export default function PreviousButton({ to = "/" }: Navigate) {
  return (
    <Link to={to} className={styles["search-nav__button--previous"]}>
      <PreviousIcon title="이전 페이지로 이동" />
    </Link>
  );
}
```

Reply:
이렇게만 바꾸면 6번라인에서 ESLint 에러가 날 텐데, optional을 지워주면 해결됩니다.
defaultProps는 함수형 컴포넌트에서 deprecated 되었습니다.
```ts
interface Navigate {
  to: string;
}
```

Reply:
근데 이렇게하면 반드시 컴포넌트 호출시 Props를 반드시 담아줘야하네요 😅
룰을 꺼버립시다

Reply:
https://github.com/BCSDLab/JJBAKSA_FRONT_END/pull/27

요거 develop 에 머지되면 rebase 한번 하시면 rule 꺼질거예요~